javascript/ql/src/experimental/Security/CWE-614/InsecureCookie.qhelp

```diff
@@ -8,7 +8,7 @@ This makes it easier for an attacker to intercept.</p>
 </overview>
 <recommendation>
 
-<p>Always set the <code>secure</code> flag to `true` on a cookie before adding it
+<p>Always set the `secure` flag to `true` on a cookie before adding it
```

Comment:
Why did you change this?
As far as I know, this will not work, but `<code>` will work.

Reply:
Sorry, I was thinking the opposite. Thanks for the feedback. I made the change.